### ISSUE TITLE
Easily exclude specific columns

### ISSUE THREAD

**Issue by lewislarsen** [motomedialab/simple-laravel-audit#5]
I have started using this package, and I think a useful addition would be a way to exclude specific columns. 

There are often a few columns that aren't particularly useful to keep track of. Perhaps something along these lines:

```
use Motomedialab\SimpleLaravelAudit\Traits\AuditableModel;

class User extends Model
{
    use AuditableModel;

    public function excludeFromAudit(): array
    {
        return [
            'created_at',
            'updated_at',
            'last_login_at',
            'remember_token',
        ];
    }
}
```

Just an idea thrown out there, let me know what you think! :)

**Comment by chrispage1** [comment]
Hey Lewis,

Completely agree on this - it's not worth logging out information that isn't particularly relevant or will be duplicated.

I'm away this week and limited internet but feel free to PR and we can collab on that - but the above looks good.

Thanks!

**Comment by lewislarsen** [comment]
#6 closes this.

**Comment by chrispage1** [comment]
Thanks for the PR @lewislarsen. Merged and tagged as v1.3.0 👍
